package.json

```diff
@@ -30,7 +30,6 @@
   },
   "devDependencies": {
     "@types/jest": "24.0.18",
-    "@types/jsonld": "^1.5.7",
```

Comment:
this was removed as its causing tests to fail, i dont think this version of types supports our jsonld version. tried to find compatible ones by downgrading a few times but still had issues

Reply:
was likely having package resolutions issues due to yarn.lock being ignored, removed my local yarn.lock and regenerated a new one now tests are passing without requiring experimental jest and types failing. updated the PR, this can be ignored